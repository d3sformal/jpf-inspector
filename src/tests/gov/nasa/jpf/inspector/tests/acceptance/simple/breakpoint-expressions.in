create breakpoint   pv and pv
create breakpoint pv or pv
create breakpoint it=cond and(mi=pos:pos)
create breakpoint ((it=cond)and(mi=pos:pos)) or pv
create breakpoint state_advanced
create breakpoint sa
create breakpoint    method_invoke = Agent   : Alf009
create breakpoint mi=mil.pentagon.target.Main:hit
create breakpoint mi=pos:pos
create breakpoint mi=mi:mi
create breakpoint mi=field_access.fr:field_access
create breakpoint mi=cg.both.cgdata.Alf:garbage_collection
create breakpoint mi=cg.both.cgdata.Cg$Runner:garbage_collection
create breakpoint    thread_scheduled = both : 0
create breakpoint ts=in:1
create breakpoint ts=out
create breakpoint    object_created = cia.Bond007
create breakpoint  objc = objc.object_created.object_released.Class.print
create breakpoint  object_released = thread.this$MyClass
create breakpoint objr=pv.property_violated.mi.array$none
create breakpoint exception_thrown=java.lang.RuntimeException
create breakpoint et=et.notify.block.me.Now
create breakpoint    field_access =   MyClass : fw
create breakpoint field=My*:x
create breakpoint fa=field:DEBUG
create breakpoint field_read=fw_fw3 : r42
create breakpoint fr          =*:init5
create breakpoint field_write =   *x*:field_write
create breakpoint fw=field_access   : a12345
create breakpoint    garbage_collection =   both
create breakpoint gc=begin
create breakpoint   gc     =end
create breakpoint    instruction_type =   any
create breakpoint it=none
create breakpoint   inst_type     =invoke
create breakpoint   it=inv
create breakpoint   it=ret
create breakpoint   it=return
create breakpoint   it=field_access
create breakpoint   it=field
create breakpoint   it=fa
create breakpoint   it=field_read
create breakpoint   it=fr
create breakpoint   it=field_write
create breakpoint   it=fw
create breakpoint   it=condition
create breakpoint   it=cond
create breakpoint   it=lock
create breakpoint   it=array
create breakpoint    position = Alf:009
create breakpoint pos=gov/nasa/jpf/inspector/JPFInspector    :  13
create breakpoint pos=gc:1
create breakpoint pos=pos:1
create breakpoint pos=field_write:2
create breakpoint pos=it:3
create breakpoint pos=inv:4
create breakpoint pos=invit:5
create breakpoint    property_violated
create breakpoint pv
create breakpoint    choice_generator =   both
create breakpoint cg=data
create breakpoint   cg     =sched
sw breakpoint