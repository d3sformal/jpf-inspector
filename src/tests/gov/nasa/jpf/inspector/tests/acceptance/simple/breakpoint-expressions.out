cmd>create breakpoint   pv and pv
cmd>create breakpoint pv or pv
cmd>create breakpoint it=cond and(mi=pos:pos)
cmd>create breakpoint ((it=cond)and(mi=pos:pos)) or pv
cmd>create breakpoint state_advanced
cmd>create breakpoint sa
cmd>create breakpoint    method_invoke = Agent   : Alf009
cmd>create breakpoint mi=mil.pentagon.target.Main:hit
cmd>create breakpoint mi=pos:pos
cmd>create breakpoint mi=mi:mi
cmd>create breakpoint mi=field_access.fr:field_access
cmd>create breakpoint mi=cg.both.cgdata.Alf:garbage_collection
cmd>create breakpoint mi=cg.both.cgdata.Cg$Runner:garbage_collection
cmd>create breakpoint    thread_scheduled = both : 0
cmd>create breakpoint ts=in:1
cmd>create breakpoint ts=out
cmd>create breakpoint    object_created = cia.Bond007
cmd>create breakpoint  objc = objc.object_created.object_released.Class.print
cmd>create breakpoint  object_released = thread.this$MyClass
cmd>create breakpoint objr=pv.property_violated.mi.array$none
cmd>create breakpoint exception_thrown=java.lang.RuntimeException
cmd>create breakpoint et=et.notify.block.me.Now
cmd>create breakpoint    field_access =   MyClass : fw
cmd>create breakpoint field=My*:x
cmd>create breakpoint fa=field:DEBUG
cmd>create breakpoint field_read=fw_fw3 : r42
cmd>create breakpoint fr          =*:init5
cmd>create breakpoint field_write =   *x*:field_write
cmd>create breakpoint fw=field_access   : a12345
cmd>create breakpoint    garbage_collection =   both
cmd>create breakpoint gc=begin
cmd>create breakpoint   gc     =end
cmd>create breakpoint    instruction_type =   any
cmd>create breakpoint it=none
cmd>create breakpoint   inst_type     =invoke
cmd>create breakpoint   it=inv
cmd>create breakpoint   it=ret
cmd>create breakpoint   it=return
cmd>create breakpoint   it=field_access
cmd>create breakpoint   it=field
cmd>create breakpoint   it=fa
cmd>create breakpoint   it=field_read
cmd>create breakpoint   it=fr
cmd>create breakpoint   it=field_write
cmd>create breakpoint   it=fw
cmd>create breakpoint   it=condition
cmd>create breakpoint   it=cond
cmd>create breakpoint   it=lock
cmd>create breakpoint   it=array
cmd>create breakpoint    position = Alf:009
cmd>create breakpoint pos=gov/nasa/jpf/inspector/JPFInspector    :  13
cmd>create breakpoint pos=gc:1
cmd>create breakpoint pos=pos:1
cmd>create breakpoint pos=field_write:2
cmd>create breakpoint pos=it:3
cmd>create breakpoint pos=inv:4
cmd>create breakpoint pos=invit:5
cmd>create breakpoint    property_violated
cmd>create breakpoint pv
cmd>create breakpoint    choice_generator =   both
cmd>create breakpoint cg=data
cmd>create breakpoint   cg     =sched
cmd>sw breakpoint
1 :  state=ENABLED hits=0 hitsTotal=0 ( property_violated ) and ( property_violated )
2 :  state=ENABLED hits=0 hitsTotal=0 ( property_violated ) or ( property_violated )
3 :  state=ENABLED hits=0 hitsTotal=0 ( instruction_type=condition ) and ( method_invoke=pos:pos )
4 :  state=ENABLED hits=0 hitsTotal=0 ( ( instruction_type=condition ) and ( method_invoke=pos:pos ) ) or ( property_violated )
5 :  state=ENABLED hits=0 hitsTotal=0 state_advanced
6 :  state=ENABLED hits=0 hitsTotal=0 state_advanced
7 :  state=ENABLED hits=0 hitsTotal=0 method_invoke=Agent:Alf009
8 :  state=ENABLED hits=0 hitsTotal=0 method_invoke=mil.pentagon.target.Main:hit
9 :  state=ENABLED hits=0 hitsTotal=0 method_invoke=pos:pos
10 :  state=ENABLED hits=0 hitsTotal=0 method_invoke=mi:mi
11 :  state=ENABLED hits=0 hitsTotal=0 method_invoke=field_access.fr:field_access
12 :  state=ENABLED hits=0 hitsTotal=0 method_invoke=cg.both.cgdata.Alf:garbage_collection
13 :  state=ENABLED hits=0 hitsTotal=0 method_invoke=cg.both.cgdata.Cg$Runner:garbage_collection
14 :  state=ENABLED hits=0 hitsTotal=0 thread_scheduled=both:0
15 :  state=ENABLED hits=0 hitsTotal=0 thread_scheduled=in:1
16 :  state=ENABLED hits=0 hitsTotal=0 thread_scheduled=out
17 :  state=ENABLED hits=0 hitsTotal=0 object_created=cia.Bond007
18 :  state=ENABLED hits=0 hitsTotal=0 object_created=objc.object_created.object_released.Class.print
19 :  state=ENABLED hits=0 hitsTotal=0 object_released=thread.this$MyClass
20 :  state=ENABLED hits=0 hitsTotal=0 object_released=pv.property_violated.mi.array$none
21 :  state=ENABLED hits=0 hitsTotal=0 exception_thrown=java.lang.RuntimeException
22 :  state=ENABLED hits=0 hitsTotal=0 exception_thrown=et.notify.block.me.Now
23 :  state=ENABLED hits=0 hitsTotal=0 field_access=MyClass:fw
24 :  state=ENABLED hits=0 hitsTotal=0 field_access=My*:x
25 :  state=ENABLED hits=0 hitsTotal=0 field_access=field:DEBUG
26 :  state=ENABLED hits=0 hitsTotal=0 field_read=fw_fw3:r42
27 :  state=ENABLED hits=0 hitsTotal=0 field_read=*:init5
28 :  state=ENABLED hits=0 hitsTotal=0 field_write=*x*:field_write
29 :  state=ENABLED hits=0 hitsTotal=0 field_write=field_access:a12345
30 :  state=ENABLED hits=0 hitsTotal=0 garbage_collection=both
31 :  state=ENABLED hits=0 hitsTotal=0 garbage_collection=begin
32 :  state=ENABLED hits=0 hitsTotal=0 garbage_collection=end
33 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=any
34 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=none
35 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=invoke
36 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=invoke
37 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=return
38 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=return
39 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=field_access
40 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=field_access
41 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=field_access
42 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=field_read
43 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=field_read
44 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=field_write
45 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=field_write
46 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=condition
47 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=condition
48 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=lock
49 :  state=ENABLED hits=0 hitsTotal=0 instruction_type=array
50 :  state=ENABLED hits=0 hitsTotal=0 position=Alf:9
51 :  state=ENABLED hits=0 hitsTotal=0 position=gov/nasa/jpf/inspector/JPFInspector:13
52 :  state=ENABLED hits=0 hitsTotal=0 position=gc:1
53 :  state=ENABLED hits=0 hitsTotal=0 position=pos:1
54 :  state=ENABLED hits=0 hitsTotal=0 position=field_write:2
55 :  state=ENABLED hits=0 hitsTotal=0 position=it:3
56 :  state=ENABLED hits=0 hitsTotal=0 position=inv:4
57 :  state=ENABLED hits=0 hitsTotal=0 position=invit:5
58 :  state=ENABLED hits=0 hitsTotal=0 property_violated
59 :  state=ENABLED hits=0 hitsTotal=0 property_violated
60 :  state=ENABLED hits=0 hitsTotal=0 choice_generator=both
61 :  state=ENABLED hits=0 hitsTotal=0 choice_generator=data
62 :  state=ENABLED hits=0 hitsTotal=0 choice_generator=scheduling